CONTRIBUTING.md

```diff
@@ -55,9 +55,9 @@ $ lerna add truffle-mycmd --scope=@truffle/core
 $ cat << EOF > core/lib/commands/truffle-mycmd/run.js
 
 module.exports = async function (options){
-    const mycmd = require("truffle-mycmd");
-    // TODO: write the run command here, something like:
-    // mycmd(options, done)
+  const mycmd = require("truffle-mycmd");
+  // TODO: write the run command here, something like:
+  // mycmd(options, done)
```

Comment:
oh also we can get rid of this callback since that is no longer part of the interface